### ISSUE TITLE
Start simulation in orbit coplanar with the Moon

### ISSUE THREAD

**Issue by siimav** [KSP-RO/RP-1#1444]

**Comment by leudaimon** [comment]
Is it by chance that the simulations currently start at 0° LAN? It's super helpful for planning interplanetary transfers!

**Comment by siimav** [comment]
Yes, all the other orbital parameters other than SMA and INC are currently set as 0.

**Comment by siimav** [comment]
I guess #2473 allows that now provided that you know the Inc and LAN numbers.
